src/sentry/models/groupsubscription.py

```diff
@@ -6,18 +6,28 @@
 from django.utils import timezone
 
 from sentry.db.models import (
-    BoundedPositiveIntegerField, FlexibleForeignKey, Model, BaseManager,
+    BaseManager, BoundedPositiveIntegerField, FlexibleForeignKey, Model,
     sane_repr
 )
 
 
 class GroupSubscriptionReason(object):
+    implicit = -1  # not for use as a persisted field value
```

Comment:
Not thrilled with this, but figured it was better than adding a random constant value elsewhere. The `CHECK` constraint on the database should prevent this from being inserted.